examples/contrib/README.md

```diff
@@ -7,5 +7,5 @@ Examples that make use of the `optax.contrib` module.
 :glob:
 :maxdepth: 1
 
-[!README.md]*
+*
```

Comment:
Remove then the comment above
<!-- include all files in the current directory except README.md -->

Reply:
good catch!